Sources/Actions/Groups.php

```diff
@@ -273,7 +273,7 @@ public function members(): void
 		Utils::$context['can_send_email'] = User::$me->allowedTo('moderate_forum');
 
 		// Load all the group moderators, for fun.
-		User::load($group->loadModerators(), User::LOAD_BY_ID, 'minimal');
+		$group->loadModerators();
```

Comment:
This is a good change to make, but it probably belongs in a separate PR.

Reply:
Well it's just a bug, I came across.  `$group->loadModerators();` returns null.  So calling `User::load()` is not necessary.  From what I can tell the original code thought that `loadModerators` returned an array of user IDs.  However `loadModerators` already calls `User::load()`

Reply:
Oh, I agree that it looks like a good and reasonable change. It's just off-topic for this PR. 🙂